.github/workflows/autoupdate-prod.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
       - name: Fetch changes
         working-directory: ./tools
         run: make fetch_openapi
+      - name: Override OpenAPI spec # TEMPORARY, WILL BE REVERTED BEFORE MERGE, USE OPEN API SPEC IN https://github.com/mongodb/atlas-sdk-go/pull/390
+        run: curl -sSfL -o openapi/atlas-api.yaml https://raw.githubusercontent.com/mongodb/atlas-sdk-go/46a77abc48d7f644c31e130fcbbd5c3f8b0ab497/openapi/atlas-api.yaml
```

Comment:
temporal commit to check with an OpenAPI spec with breaking changes with current spec

Reply:
reverted